packages/angular/src/generators/storybook-configuration/storybook-utilities.spec.ts

```diff
@@ -1,15 +1,16 @@
 import { joinPathFragments, Tree, writeJson } from '@nx/devkit';
-import {
-  overrideCollectionResolutionForTesting,
-  wrapAngularDevkitSchematic,
-} from '@nx/devkit/ngcli-adapter';
+import { overrideCollectionResolutionForTesting } from '@nx/devkit/ngcli-adapter';
```

Comment:
This helper and its usage shouldn't be needed anymore. It was only needed due to the schematic usage.